rc/core/comment.kak

```diff
@@ -50,7 +59,8 @@ hook global BufSetOption filetype=haml %{
 
 hook global BufSetOption filetype=html %{
     set buffer comment_line ''
-    set buffer comment_block '<!--:-->'
+    set buffer comment_block_begin '<lt>!--'
```

Comment:
I also originally avoided doing this to make things simpler even if that meant spawning a process to escape the separators later on.